templates/404.html

```diff
@@ -0,0 +1,21 @@
+{% extends "_base/base.html" %} {% block title %}404: Page not found | Juju{% endblock %} {%
```

Comment:
I feel maybe `_includes/` or `_layout/` is a more appropriate name than `_base` for the folder. The repetition of `_base/base.html` is a bit jarring.

Reply:
I see now that you will have copied this from jp.ubuntu.com. But I still don't like it and suggest changing it ;)

Reply:
I figure we should probably use the same wrapper as jaas.ai in the end. I'll see what we can do about bringing these projects together on the wrapper.

Reply:
Sorry what do you mean? jaas.ai doesn't have a `templates/_base` folder. That's what I was suggesting you rename in this case.

But if you mean the markup, I still think that while the GUI and jaas.ai are separate domains/subdomains, the GUI should have error pages that appear consistent with the GUI's look and feel. Maybe that just means a blank page with a message in the middle? Similar to how the loading screen looks. That would be fine.